drgn_tools/task.py

```diff
@@ -161,3 +169,93 @@ def task_cpu(task: drgn.Object) -> int:
     if has_member(task, "cpu"):
         return task.cpu.value_()
     return task_thread_info(task).cpu.value_()
+
+
+def translate_nanoseconds(task: Object) -> str:
+    """
+    :returns: conversion of nanoseconds to [dd hh:mm:ss.ms] format
+    """
+    nanosecs = task_lastrun2now(task)
+    secs = nanosecs_to_secs(nanosecs)
+    time = str(datetime.timedelta(seconds=secs))
+    return time
```

Comment:
This returns a string like `83 days, 19:15:12.102000` whereas the crash command outputs in format `0 00:00:00.000`. While I'm not normally a stickler for _exact_ compatibility, it seems that it's important to me to output things roughly the same, since the table alignment would look strange otherwise. You'll just have to format the string yourself to get that.